data/examples/simba.cfg

```diff
@@ -95,6 +95,15 @@ min_charging_time = 0
 # NOTE: else clause is a MUST! The buffer time is deducted off of the planned standing time.
 # It may resemble things like delays and/or docking procedures (default: 0)
 default_buffer_time_opps = 0
+# Buffer time in min at depot stations (default: 0).
+default_buffer_time_deps = 0
+
+# Assign vehicles from depots using an "adaptive" strategy which uses vehicles with socs above the consumption of the
+# rotation or use "fixed_recharge" which uses vehicles for rotations if the vehicles reached the soc values defined with
+# min_recharge_deps_oppb or depb. (default: "adaptive").
```

Comment:
config files have no line length restriction, no need to have it break in the middle of the sentence